apollo-compiler/src/test/graphql/com/example/antlr_tokens/java/operationBased/antlr_tokens/TestQuery.java.expected

```diff
@@ -111,8 +110,13 @@ public class TestQuery implements Query<TestQuery.Data> {
   }
 
   @Override
-  public List<CompiledSelection> selections() {
-    return TestQuerySelections.root;
+  public CompiledField rootField() {
+    return new CompiledField.Builder(
+      "data",
+      com.example.antlr_tokens.type.Query.type
```

Comment:
Nitpick: would it be possible to have it imported to avoid the FQN?

Reply:
This is kotlinpoet land. 
Maybe possible, maybe not. Overall, KotlinPoet focuses at making correct code. Making the look nice is not the primary goal. We could use something like ktlint on top of KotlinPoet if we really wanted to.

Reply:
Fair enough!